Makefile

```diff
@@ -113,14 +118,48 @@ UNIT_DIR        := unit
 UNITTESTS_DIR   := $(TESTS_DIR)/$(UNIT_DIR)
 
 # Symbols needed to integrate with LineOfCode (LOC) package
-LOCPACKAGE      := LineOfCode/loc
+LOC_ROOT        := LineOfCode
```

Comment:
Most of the changes hereonwards are to do with awareness of LOC-ELF encoding scheme.

And, to pick-up different build code-paths depending on LOC-encoding scheme in effect.